html/diff_page.go

```diff
@@ -177,45 +178,53 @@ func (c *DiffPage) WriteHTMLTo(w io.Writer) (int64, error) {
 		precalculatedComparisons = append(precalculatedComparisons, comparison)
 	}
 
+	args := os.Args
```

Comment:
As mentioned in the other diff, you should't be passing the params here or like this. Pass them through `DiffPage` if you need access to them.